File: GerenciadorDeMilhas.Domain/Cliente.cs
using System;
using System.Collections.Generic;

namespace GerenciadorDeMilhas.Domain
{
    public class Cliente
    {
        public int Id { get; set; }
        public string Nome { get; set; }

        public DateTime Data { get; set; }

        public string CPF { get; set; }

        public string RG { get; set; }

        public string Endereco { get; set; }

        public string Bairro { get; set; }

        public string Cidade { get; set; }

        public string Numero { get; set; }
        
        public string UF { get; set; }

        public string Pais { get; set; }
        
        public string Complemento { get; set; }

        public string Skype { get; set; }

        public bool Status { get; set; }

        public List<Email> Emails { get; set; }
        
        public List<Telefone> Telefones { get; set; }

        public List<AcessoCliente> AcessoClientes { get; set; }

        public List<Historico> Historicos { get; set; }

        public ContaCorrente ContaCorrente { get; set; }
    }
}

File: GerenciadorDeMilhas-App/src/app/_models/Cliente.ts
import { Email } from './Email';
import { Telefone } from './Telefone';
import { AcessoCliente } from './AcessoCliente';
import { Historico } from './Historico';
import { ContaCorrente } from './ContaCorrente';

export interface Cliente {

    id: number;

    nome: string;

    data: Date;

    cpf: string;

    rg: string;

    endereco: string;

    bairro: string;

    cidade: string;

    numero: string;

    uf: string;

    pais: string;

    complemento: string;

    skype: string;

    status: boolean;

    emails: Email[];

    telefones: Telefone[];

    acessoClientes: AcessoCliente[];

    historicos: Historico[];

    contaCorrente: ContaCorrente;
}
